code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -181,20 +181,20 @@
 
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/taser
+	equip_cooldown = 0.8 SECONDS
 	name = "\improper PBT \"Pacifier\" Mounted Taser"
 	icon_state = "mecha_taser"
 	origin_tech = "combat=3"
 	energy_drain = 20
-	equip_cooldown = 8
```

Comment:
I personally would leave this var around this place instead of coming before the name

Reply:
half of the guns have a cooldown before and the other half after, I thought I'd just lump it all together.